sdk/core/core-tracing/src/globals.d.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Instrumenter } from "./interfaces";
+
+// eslint-disable-next-line @azure/azure-sdk/ts-no-namespaces
+declare global {
+  // eslint-disable-next-line no-var
```

Comment:
Really not sure about this - would love any suggestions such that I can do `globalThis.instrumenterImplementation = something`

Reply:
I think using `Symbol.for()` and using the resulting symbol as a property name will work better than using some normal identifier